.drone.yml

```
@@ -1,168 +1,134 @@
 ---
 kind: pipeline
 type: kubernetes
-name: lint
-
+name: build
 trigger:
   event:
     - push
+  branch:
+    exclude:
+      - main
 
 steps:
-- name: lint bash
-  image: koalaman/shellcheck-alpine
-  commands:
-    - shellcheck *.sh */*.sh */*/*.sh
-  depends_on:
-    - clone
+  - name: build & push
+    image: plugins/docker
+    settings:
+      registry: quay.io
+      repo: quay.io/ukhomeofficedigital/hocs-toolbox
+      tags:
+        - ${DRONE_COMMIT_SHA}
+    environment:
+      DOCKER_PASSWORD:
+        from_secret: QUAY_ROBOT_TOKEN
+      DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
+    depends_on:
+      - clone
 
 ---
 kind: pipeline
 type: kubernetes
-name: build
+name: deploy
+trigger:
+  event:
+    - promote
+
+steps:
+  - name: fetch and checkout
+    image: alpine/git
+    commands:
+      - git fetch --tags
+      - git checkout $${VERSION}
+
+  - &deploy
+    name: cs-prod
+    image: quay.io/ukhomeofficedigital/kd
+    environment:
+      VERSION: ${DRONE_TAG}
+      KUBE_TOKEN:
+        from_secret: hocs_toolbox_prod
+    commands:
+      - cd kube
+      - ./deploy.sh
+    depends_on:
+      - fetch and checkout
+    when:
+      target:
+        include:
+          - "*-prod"
+
+  - <<: *deploy
+    name: wcs-prod
 
-platform:
-  os: linux
-  arch: amd64
 
+---
+kind: pipeline
+type: kubernetes
+name: build tag
 trigger:
+  event:
+    - tag
   branch:
     - main
-  event:
-    - push
 
 steps:
-  - name: build & push safe
+
+  - name: build & push tag main
     image: plugins/docker
-    depends_on:
-      - clone
     settings:
       registry: quay.io
       repo: quay.io/ukhomeofficedigital/hocs-toolbox
-      target: safe
       tags:
-        - build_${DRONE_BUILD_NUMBER}
-        - ${DRONE_COMMIT_SHA}-safe
+        - ${DRONE_TAG}
+        - ${DRONE_COMMIT_SHA}
```

```suggestion
        - ${DRONE_COMMIT_SHA}
```
Consider removing the SHA tag from the `git tag` and put in the `git push`. (I guess ideally we'd pull the existing SHA tag from the `git push` pipeline and re-tag it with the `git tag` to avoid building twice, but that's not trivial.)